src/style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+}
+
+.container {
```

Comment:
actually, you just dont need this container in this task

Reply:
deleted